components/reusable-components/login-form/SubmitLoginForm.tsx

```diff
@@ -7,13 +7,13 @@ export interface LoginFormData {
 type LoginFormResponse = any;
 
 export const submitLoginForm = async (formValues: LoginFormData): Promise<LoginFormResponse> => {
-  const url = process.env.NEXT_PUBLIC_API_BASE_URL + '/login'!;
+  const url = `${process.env.NEXT_PUBLIC_API_BASE_URL}/content-login`;
```

Comment:
check the correct url here: https://api.learnhub.mk/docs/#authentication-POSTcontent-login

Reply:
also you should use the login from authContext